collections/_article/godot-developer-fund.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing the new Godot Development Fund"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us keep the project alive.
```

Comment:
```suggestion
We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
```

I'm not sure my suggestion is perfect. But I do think we should use more subtle language than "help us keep the project alive". While technically it has the same meaning as my suggestion, it also makes it sound like the project is out of funds (which was a common misconception from our last funding post). 

Personally, I think we should be highlighting 2 things:
1. Longevity (we need to ensure that development pace is maintained as long as possible)
2. Independence. The more funding we have from individuals, the less risk there is that Godot is "selling out". This is more of a public perception thing than an actual problem.

Reply:
Ah, I see you do that more below.